src/chip-typeahead/index.tsx

```diff
@@ -255,10 +234,17 @@ export const ChipTypeahead = factory(function ChipTypeahead({
 				onBlur={() => icache.set('focused', false)}
 				classes={{
 					'@dojo/widgets/text-input': {
-						inputWrapper: [themeCss.inputWrapper],
 						input: [themeCss.input],
 						wrapper: [themeCss.wrapper],
-						leading: [themeCss.inputLeading]
+						leading: [themeCss.inputLeading],
+						leadingWrapper: [themeCss.inputLeadingWrapper],
+						defaultKind: [themeCss.defaultKind],
+						outlinedKind: [themeCss.outlinedKind],
+						labelWrapper: [themeCss.inputLabel],
+						label: [themeCss.label, chips.length ? themeCss.labelActive : undefined],
```

Comment:
`chips.length && themeCss.labelActive`

Reply:
not sure it liked that when i tried before